features/fxa-migration/README.md

```diff
@@ -0,0 +1,38 @@
+
+AMO to Firefox Account Migration UX Specs
+=========================================
+
+So that I can reduce the numbers of logins I have, as a Mozilla Add-ons user, I want to switch from using my AMO login and start using a Firefox Account.
+
+## User Flow Diagram
+Red line denotes common post-migration login experience.
+![User Flow Diagram](https://www.lucidchart.com/publicSegments/view/c45f10ff-5084-407d-8322-cbc8b6cd25be/image.png)
```

Comment:
If you put the image in git rather than linking to it then we can diff the changes you make. Downside being you need to update the image in git when that happens but it's a good way to let us know there have been changes.